metadata/service/aggregator/single.yml

```diff
@@ -6,3 +6,4 @@ parameters:
   heka:
     aggregator:
       enabled: true
+      timezone: ${_param:linux_timezone}
```

Comment:
The issue is that the "sane" default is [different](https://github.com/tcpcloud/salt-formula-linux/blob/a38a3ccf1ce45ba99ee0d5d2a563ea56e4d3e5fc/metadata/service/system/init.yml#L14) in the linux formula :) I'd use `salt['pillar.get']('linux:system:timezone')` to avoid introducing a new configuration parameter.